client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1772,7 +1772,6 @@
   "report.requested-from": "Requested from",
   "report.requested-qty": "Requested Qty",
   "report.requisition-category-message": "Requisition Category: Category Value",
-  "report.satisfaction-rate-per-program": "Satisfaction rate per program",
```

Comment:
Removing since report name for custom reports should always be in the language of the country requesting and will show as such